random test to be modified later

midizzz opened the pull request.

midizzz committed:
random test to be modified later

midizzz committed:
random test to be modified later

dragon-foobar commented on content/vault/.obsidian/workspace.json:
```diff
@@ -13,7 +13,7 @@
               "type": "markdown",
               "state": {
-                "file": "SatoshiPlace.md",
+                "file": "BlogPost1.md",
```
Not sure why you need to do this?

dragon-foobar commented on content/vault/BlogPost1.md:
```diff
@@ -0,0 +1,29 @@
+
+# Welcome!
+
+
```
I like what you've done here, but maybe you could customise the blog to make it unique ?

dragon-foobar reviewed (changes requested):
hmm.. good first try dude. Maybe you could write an article that isn't the same as welcome.

midizzz committed:
MultiSig Magic Trust Machine

I think this is finished now, just needs a quick proof read by someone Else

midizzz committed:
MultiSig Magic Trust Machine

I think this is finished now, just needs a quick proof read by someone Else

unspentspace-admin commented on content/vault/BlogPost1.md:
```diff
@@ -9,21 +9,58 @@ tags:
 ---
 
-# Welcome!
+# MultiSig Magic Trust Machine
```
Nice title!

unspentspace-admin commented:
Awesome, man. Just a few spelling/grammar things. Maybe easier if I add a commit on? @midizzz

midizzz commented:
> Awesome, man. Just a few spelling/grammar things. Maybe easier if I add a commit on? @midizzz 

Yeah. I want to submit again. Then you can proof it. Still to ruff. Just testing the method. Seems it works

midizzz committed:
Ok now

Phewf there was a lot of trash to tidy. Still needs your quality reviews however

midizzz committed:
Ok now

Phewf there was a lot of trash to tidy. Still needs your quality reviews however

midizzz commented:
ok, not sure if I am typing in the right place.. It looks like you have seen this, maybe there's somthing about a tag "satoshis place"? no idea. There must be some typos etc?

dragon-foobar commented:
@midizzz there are a couple of typos still, but that's not the main issue. The main issue is that your branch is now not in harmony with the main branch. You need to merge dev branch into your branch. We can do a screen share or something.

midizzz commented:
tbh at the moment if feels like the communication here falls into black hole

midizzz committed:
pro act

something

midizzz committed:
pro act

something

midizzz committed:
Merge branch 'main' into first-test

nickfarrow committed:
Wording and grammar

nickfarrow commented:
Rebased and removed excess changes (DS_STORE and obsidian changes)

I also did a proof read, with a solid amount of wording changes and grammar fixes. Inspiring post, I want to start a peer multisig now. Please check I have accurately captured your words 0b117ce4a05ab624553914c996de2242165877a4

I suggest to hit "Squash and Merge" for this one so we can keep the commits clean

midizzz committed:
resubmitted blog for review

midizzz commented:
Thanks @nickfarrow .. Oh no. I have also reviewed this cutting it down a lot and have been trying to resubmit. I am just getting used to the Obsidian to Github flow here.. I better have a look at your edits in context of the new revision named Magic MultiSig Trust Machine.

midizzz committed:
Merge branch 'first-test' of https://github.com/unspentspace/unspent.space into first-test

midizzz committed:
Delete CPHcss

fixing amateur moves
